src/Saas.Application/Saas.Application.Web/Views/Shared/_LoginPartial.cshtml

```diff
@@ -0,0 +1,26 @@
+﻿@using Microsoft.Identity.Web
+@if (User.Identity.IsAuthenticated)
```

Comment:
Fix warning

Reply:
Per discussion, these are symptoms of a wider refactor that will affect the Signup app as well and will be handled in their own PR.